atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -23,7 +23,7 @@
 from atomic_reactor.plugins.pre_check_and_set_rebuild import is_rebuild
 from atomic_reactor.util import (OSBSLogs, get_parent_image_koji_data, get_manifest_media_version,
                                  is_manifest_list)
-from atomic_reactor.utils.koji import get_buildroot as koji_get_buildroot
+from atomic_reactor.utils.koji import get_buildroot as koji_get_buildroot, get_maven_components
 from atomic_reactor.utils.koji import get_output as koji_get_output
 from atomic_reactor.utils.koji import (
```

Comment:
@hjmodi; Nit-picky, but I would much prefer that `get_maven_components` be imported in this block, e.g.

```python
from atomic_reactor.utils.koji import (
    generate_koji_upload_dir, add_custom_type,
    get_source_tarball_output, get_remote_source_json_output,
    get_maven_components
)
```

tl;dr `get_buildroot` and `get_output` are being aliased very purposefully, the other modules (the ones in the block) are simply straight imports.

Reply:
Updated